### ISSUE TITLE
Can't get jetcard image (option 1) to do anything on Jetson Nano

### ISSUE THREAD

**Issue by michael714** [NVIDIA-AI-IOT/jetcard#25]
I have tried downloading the image and installing it on multiple micro SD cards with no luck.  I can easily get the standard Jetson Nano image to work and the jetbot image works fine, but not the jetcard image.  I have used SD cards that worked fine with other images to try the jetcard image with no luck.  When I use the jetcard image, I get no indication that anything is happening on the Jetson Nano.  No boot messages, nothing.

**Comment by jalalirs** [comment]
Me too. I have been trying to boot it using the Jetcard image it doesn't work. When I connect my micro-USB to the laptop, nothing happens. I have tried every possible scenario for the power supply and still no luck. Recently, I generated the hash for the image using Win32 Disk Image, and it doesn't look like it is equal to what is provided. 

The hash generated by Win32Disk Imager is:
20b044d58ba0480539a94649faf27f27
The expected hash
f7b635a651e4a2228e3812360cce74e3

Please advice...

**Comment by jalalirs** [comment]
> I have tried downloading the image and installing it on multiple micro SD cards with no luck. I can easily get the standard Jetson Nano image to work and the jetbot image works fine, but not the jetcard image. I have used SD cards that worked fine with other images to try the jetcard image with no luck. When I use the jetcard image, I get no indication that anything is happening on the Jetson Nano. No boot messages, nothing.

Did you solve this issue?

**Comment by kogantiharsha** [comment]
> Me too. I have been trying to boot it using the Jetcard image it doesn't work. When I connect my micro-USB to the laptop, nothing happens. I have tried every possible scenario for the power supply and still no luck.
> Please advice...
Yes, it is not showing up anything on hdmi monitor. But after initial setup according to the nvidia jetracer repository it works just fine in remote terminal.

**Comment by cyrilkmathew** [comment]
I have tried to install jetcard it on micro SD cards (Option 1 - Download JetCard directly). unfortunately, it is not booting. It shows only Nvidia flash screen.

**Comment by andur** [comment]
SOLUTION:
The Firmware of my Jetson 4GB has been modified by the latest Jetpack Image!
Therefore it doesn't boot from old Images like this one. The Image is OBSOLETE.
You have to get the latest Jetpack-Image, Boot your Jetson and rebuild this Solution from Scratch by the Git-Repository
Obviously, there will be lots of errors, because of changed dependencies over the last 2 years, but thats another story...
